cognite/neat/graph/queries/_base.py

```diff
@@ -112,25 +112,34 @@ def describe(
         property_values: dict[str, list[str]] = defaultdict(list)
 
         for subject, predicate, object_ in cast(list[ResultRow], self.graph.query(f"DESCRIBE <{instance_id}>")):
-            # We cannot include the RDF.type in case there is a neat:type property
-            # or if the object is empty
-            if predicate != RDF.type and object_.lower() not in [
+            if object_.lower() not in [
                 "",
                 "none",
                 "nan",
                 "null",
             ]:
                 # we are skipping deep validation with Pydantic to remove namespace here
                 # as it reduce time to process triples by 10-15x
-                identifier, property_, value = cast(  # type: ignore[misc]
-                    (str, str, str),
-                    remove_namespace_from_uri(*(subject, predicate, object_), validation="prefix"),
+                identifier, value = cast(  # type: ignore[misc]
+                    (str, str),
+                    remove_namespace_from_uri(*(subject, object_), validation="prefix"),
                 )  # type: ignore[misc, index]
 
-                if property_renaming_config:
-                    property_ = property_renaming_config.get(property_, property_)
+                # use-case: calling describe without renaming properties
+                # losing the namespace from the predicate!
+                if not property_renaming_config:
+                    property_values[remove_namespace_from_uri(predicate, validation="prefix")].append(value)
```

Comment:
Isn't there a chance that you have a `my_namespace:type` and `rdf:type` that will end up both being named `type`?